src/cpp/client.cpp

```diff
@@ -850,7 +850,10 @@ void Client::_append_dataset_metadata_commands(CommandList& cmd_list,
                                  "a DataSet into the database that "\
                                  "does not contain any fields or "\
                                  "tensors.");
-    // SingleKeyCommand* cmd = new SingleKeyCommand();
+    SingleKeyCommand* del_cmd = cmd_list.add_command<SingleKeyCommand>();
+    del_cmd->add_field("DEL");
+    del_cmd->add_field(meta_key, true);
```

Comment:
We're creating a command and then doing nothing with it here -- doesn't seem to be intended

Reply:
Was this test code that didn't get deleted?

Reply:
``del_cmd`` is being added to the ``cmd_list`` to be executed

Reply:
I'm not following that "nothing is done with it".  Can you elaborate?

Reply:
My bad, the semantics tripped me up again.